### PR TITLE
Extend "S3 buckets with SSE-KMS" documentation 

### PR DESCRIPTION
**Pull request by drernie**
# Description

Add new sections `technical-reference.md` about explicit Allow permissions required by KMS

In order for Quilt to access and index buckets encrypted with SSE-KMS, you must do three things:

1. Add KMS Key Usage to Quilt Permission Boundary
2. Add Quilt Principals to KMS Key Policy [this was the old section]
3. Add KMS Key Access to Quilt Role

### CODE REVIEW

**Review by sir-sigurd — COMMENTED** [review]
I think you forgot to commit images.

**Comment by drernie** [comment]
> I think you forgot to commit images.

Whoops! Thanks. Fixed.

**Comment by sir-sigurd** [comment]
> > I think you forgot to commit images.
> 
> Whoops! Thanks. Fixed.

But what about admin-policy-*.png?

**Comment by drernie** [comment]
> > > I think you forgot to commit images.
> > 
> > 
> > Whoops! Thanks. Fixed.
> 
> But what about admin-policy-*.png?

Ah! Forgot to pull everything over from old branch. Pushed now
`git checkout athena-config -- docs/imgs`

File: docs/technical-reference.md [hunk 1]
```diff
@@ -420,6 +453,31 @@ similar to the following to the KMS key policy:
...
+    ],
+    "Resource": [
+      "arn:aws:kms:us-west-2:111122223333:key/1234abcd-12ab-34cd-56ef-1234567890ab",
+      "arn:aws:kms:us-west-2:111122223333:key/0987dcba-09fe-87dc-65ba-ab0987654321"
+    ]
+  }
+}
+```
+
+You can now create a Quilt Policy from this policy (using the Catalog's admin panel, or `quilt3.admin.create_policy`).
```
Comment (sir-sigurd) [inline docs/technical-reference.md:477]: There is no `quilt3.admin.create_policy()` and actually `quilt3.admin` is broken needs to be fixed or removed. There is conversion about that somewhere in Slack and maybe ticket in Asana.

Comment (drernie) [reply docs/technical-reference.md:477]: Ouch. Okay, I will just remove those references and ensure there's an Asana ticket.

Comment (drernie) [reply docs/technical-reference.md:477]: Created new Task, assigned for Rob to prioritize https://app.asana.com/0/1201334960547652/1202896362990186/f

Comment (drernie) [reply docs/technical-reference.md]: Removed.

File: docs/Catalog/Admin.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ the panel via a dropdown menu under username in the navbar.
 
 The admin page is only accessible to designated administrators. The first admin
 is set during the CloudFormation installation. Subsequent admins may be designated
-through the panel. Only admins may create other admins. 
+through the panel. Only admins may create other admins.
```
Comment (akarve) [inline docs/Catalog/Admin.md:10]:
```suggestion
through the panel. Only admins can create other admins.
```

File: docs/Catalog/Admin.md [hunk 3]
```diff
@@ -65,10 +70,78 @@ The resulting permission set is equivalent to a union of all permissions
...
+1. Login to your Quilt instance at, e.g. https://quilt.mycompany.com
+2. Click on "Admin Settings" in the upper right, under your Profile name
+3. Scroll down to the "Roles" section on the bottom
+4. Click "+" to define a new role
+5. Name it `UserReadQuiltBucket`
+6. Save
+
+You can also create a `UserReadWriteQuiltBucket` if you want grant full permissions
+
+#### B. Display existing policies in AWS Console
```
Comment (akarve) [inline docs/Catalog/Admin.md:91]: For now let's eliminate the user finding and attaching Quilt policies managed by the stack. Please only document creating and attaching Quilt policies, made in Quilt, via the UI.

**Review by akarve — CHANGES_REQUESTED** [review]

File: docs/Catalog/Admin.md (hunk 3 above)
Comment (drernie) [reply docs/Catalog/Admin.md:91]: @akarve Do you still feel strongly that this revision should NOT allow people to create new Source=Quilt roles that leverage the existing Read/ReadWrite Source=Custom policies?

**Comment by drernie** [comment]
Closing. Will re-open with new branch on just KMS edits.

**Pull request closed without merging**